docs/Homebrew-Governance.md

```diff
@@ -54,7 +54,7 @@
 
 1. The governance and operation of Homebrew is determined by the PLC. The PLC will represent Homebrew in all dealings with the Software Freedom Conservancy (SFC). The PLC has final authority in all matters related to the operation of Homebrew, and will comply with all Software Freedom Conservancy policies.
 
-2. The PLC consists of five members. Committee members are elected by majority vote of Homebrew members. Each PLC member will serve a term of two years or until the member's successor is elected. Any sudden vacancy in the PLC will be filled by the usual procedure for electing PLC members at the next general meeting, typically the next AGM.
+2. The PLC consists of five members. Committee members are elected by Homebrew members in a Meek Single Transferable Vote (STV) election using the Droop quota formula. Each PLC member will serve a term of two years or until the member's successor is elected. Any sudden vacancy in the PLC will be filled by the usual procedure for electing PLC members at the next general meeting, typically the next AGM.
```

Comment:
```suggestion
2. The PLC consists of five members. Committee members are elected by Homebrew members in a Meek Single Transferable Vote election using the Droop quota. Each PLC member will serve a term of two years or until the member's successor is elected. Any sudden vacancy in the PLC will be filled by the usual procedure for electing PLC members at the next general meeting, typically the next AGM.
```
No need to define STV since it's not used later. `formula` is redundant.